src/SUFST/inc/telemetry_protocol.h

```diff
@@ -5,30 +5,6 @@
 #include <can_database.h>
 
 #define TABLE_SIZE 23
```

Comment:
I'd give this macro a more specific name to avoid the chance of collision. What table is this the size of?

Reply:
This is same length as table provided by can-defs. Shall we add the constant in can-defs to be accessible by user?

Reply:
Yes that sounds like a good way to do it 👍

Reply:
Created Issue [#16](https://github.com/sufst/can-defs/issues/16) for it.